src/main/java/frc/robot/utilities/pid/TrigonPIDFController.java

```diff
@@ -0,0 +1,85 @@
+package frc.robot.utilities.pid;
+
+import edu.wpi.first.wpilibj.controller.PIDController;
+
+public class TrigonPIDFController extends PIDController implements PIDFConfigurable {
+    private double v;
+    private double s;
```

Comment:
why not `kV' and `kS` ?

Reply:
because in the past it was f and not kF. blame the person who wrote it then

Reply:
I'm requesting changes, not blaming anyone